.env.example

```
@@ -1,12 +1,7 @@
-PUBLIC_ADDRESS1=
-PRIVATE_KEY1=
-ERC20_TOKEN_ADDRESS=
+# All variables are optional — `yarn test:integration` runs with sensible defaults.
 
-PUBLIC_ADDRESS2=
-PUBLIC_ADDRESS3=
+# ---------- Fork RPCs ----------
+# Override the default public RPCs used by anvil to fork each chain.
+# Defaults: ethereum-rpc.publicnode.com, optimism-rpc.publicnode.com, mainnet.base.org
+ETHEREUM_RPC=
+OPTIMISM_RPC=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)